### PR TITLE
Da dc56 11

### PR DESCRIPTION
**Pull request by Misca1234**
This is DA56-11 and DC56-11

kicad-library push branch is 
https://github.com/KiCad/kicad-library/pull/1458

Displays_7-Segment.pretty push branch is 
https://github.com/KiCad/Displays_7-Segment.pretty/pull/11

packages3D push branch is 
https://github.com/KiCad/packages3D/pull/97

packages3D-source push branch is 
https://github.com/KiCad/packages3D-source/pull/31

DA56-11CGKWA 
http://www.kingbright.com/attachments/file/psearch/000/00/00/DA56-11CGKWA(Ver.16A).pdf

DA56-11EWA 
http://www.kingbrightusa.com/images/catalog/SPEC/DA56-11EWA.pdf

DA56-11GWA
http://www.kingbrightusa.com/images/catalog/SPEC/DA56-11GWA.pdf

DA56-11SEKWA 
http://www.kingbright.com/attachments/file/psearch/000/00/00/DA56-11SEKWA(Ver.9A).pdf

DA56-11SURKWA 
http://www.kingbright.com/attachments/file/psearch/000/00/00/DA56-11SURKWA(Ver.11A).pdf

DA56-11SRWA 
http://www.kingbrightusa.com/images/catalog/SPEC/DA56-11SRWA.pdf

DA56-11SYKWA 
http://www.kingbright.com/attachments/file/psearch/000/00/00/DA56-11SYKWA(Ver.11A).pdf

DA56-11YWA 
http://www.kingbrightusa.com/images/catalog/SPEC/DA56-11YWA.pdf

DC56-11CGKWA 
http://www.kingbright.com/attachments/file/psearch/000/00/00/DC56-11CGKWA(Ver.7A).pdf

DC56-11EWA 
http://www.kingbrightusa.com/images/catalog/SPEC/DC56-11EWA.pdf

DC56-11GWA
http://www.kingbrightusa.com/images/catalog/SPEC/DC56-11GWA.pdf

DC56-11SEKWA 
http://www.kingbright.com/attachments/file/psearch/000/00/00/DC56-11SEKWA(Ver.7A).pdf

DC56-11SRWA 
http://www.kingbrightusa.com/images/catalog/SPEC/DC56-11SRWA.pdf

DC56-11SURKWA 
http://www.kingbright.com/attachments/file/psearch/000/00/00/DC56-11SURKWA(Ver.8A).pdf

DC56-11SYKWA 
http://www.kingbright.com/attachments/file/psearch/000/00/00/DC56-11SYKWA(Ver.7A).pdf

DC56-11YWA 
http://www.kingbrightusa.com/images/catalog/SPEC/DC56-11YWA.pdf

bild

### CODE REVIEW

**Comment by poeschlr** [comment]
Merge required.

**Comment by Misca1234** [comment]
Merge fixed

**Comment by poeschlr** [comment]
Something went wrong with your merge. I get an error message when i try to open the library.

File: library/display.lib [hunk 1]
```diff
@@ -1174,6 +1329,159 @@ X g 16 -850 -300 300 R 50 50 1 1 I
...
+X DIG2_B 10 600 200 100 L 50 50 1 1 I
+X DIG2_A 11 600 300 100 L 50 50 1 1 I
+X DIG2_F 12 600 -200 100 L 50 50 1 1 I
+X DIG2_CC 13 600 -500 100 L 50 50 1 1 I
+X DIG1_CC 14 -600 -500 100 R 50 50 1 1 I
+X DIG1_B 15 -600 200 100 R 50 50 1 1 I
+X DIG1_A 16 -600 300 100 R 50 50 1 1 I
+X DIG1_G 17 -600 -300 100 R 50 50 1 1 I
+X DIG1_F 18 -600 -200 100 R 50 50 1 1 I
+#
```
Comment (poeschlr) [inline library/display.lib]: I think ENDDRAW and ENDDEF are missing here

**Comment by Misca1234** [comment]
Yes, my online hand merge failed, but is now corrected

**Comment by SchrodingersGat** [comment]
@poeschlr are you happy with this branch now?

**Comment by SchrodingersGat** [comment]
Yep merge seems to have fixed it. Thanks for your contribution

**Pull request merged**